conan/cache/db/recipes_table.py

```diff
@@ -53,38 +53,27 @@ def get(self, ref: ConanReference):
             raise ConanReferenceDoesNotExistInDB(f"No entry for recipe '{ref.full_reference}'")
         return self._as_dict(self.row_type(*row))
 
-    def save(self, path, ref: ConanReference, timestamp):
-        # we set the timestamp to 0 until they get a complete reference, here they
-        # are saved with the temporary uuid one, we don't want to consider these
-        # not yet built packages for search and so on
+    def create(self, path, ref: ConanReference, timestamp):
```

Comment:
Try to explicit say this is only a "creation" scenario, not possible an "update" calling it. Reference should not exist in DB yet